packages/web/src/features/entitlements/server.ts

```
@@ -25,29 +44,62 @@ export const getPlan = (): Plan => {
+        if (licenseKey.customEntitlements) {
+            return "self-hosted:enterprise-custom";
+        }
+        return licenseKey.seats === SOURCEBOT_UNLIMITED_SEATS ? "self-hosted:enterprise-unlimited" : "self-hosted:enterprise";
     }
 
     return "oss";
 }
 
+export const getSeats = (): number => {
+    const licenseKey = getLicenseKey();
+    return licenseKey?.seats ?? SOURCEBOT_UNLIMITED_SEATS;``
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)